src/egon/data/datasets/scenario_parameters/parameters.py

```diff
@@ -786,7 +786,7 @@ def gas(scenario):
         interest_rate = 0.07  # [p.u.]
 
         parameters = {
-            "main_gas_carrier": "H2",
+            "main_gas_carrier": "CH4",
```

Comment:
Could you explain why you changed the main_gas_carrier for eGon100RE?

Reply:
Thanks for the comment. It is a mistake. It is changed in aa1d9f22d8212a93377e3ae732f87b184e5005eb.